author.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './item'
+
+class Author < Item
+  attr_reader :id, :items
+  attr_accessor :first_name, :last_name
+
+  def initialize(first_name, last_name)
+    super()
+    @first_name = first_name
+    @last_name = last_name
+    @id = id || Random.rand(1...10_000)
+    @items = []
+  end
+
+  def add_item(item)
+    item.add_author(self)
+  end
```

Comment:
You may try to test the information that you're receiving to avoid errors.
For exemple
`(item.is_a?(Item) ) || (@items << item)`

Reply:
Thanks!